docs/pages/toolchain/katana/interact.md

```diff
@@ -0,0 +1,377 @@
+### Install starkli
+
+Open a new terminal
+
+```bash
+curl https://get.starkli.sh | sh
+starkliup
+```
+
+You can check your installation by running `starkli --version`,then you will get the starkli version.
+
+### Install katana
+
+```bash
+curl -L https://install.dojoengine.org | bash
+dojoup -v 0.6.0-alpha.3
+```
+
+You can check your installation by running `katana --version`,then you will get the katana version.
+
+### Basics of Katana and Starkli
+
+#### katana sequencer
+
+```console
+katana --disable-fee
+```
+
+After starting the node, a list of accounts will be automatically generated and deployed.
+
+#### Starkli built-in accounts and configuration
+
+Starkli supports a list of built-in accounts for katana. These built-in accounts are for local development and no one should be using them for anything serious anyways. For example the address of the `Katana-0` built-in account is `0x6162896d1d7ab204c7ccac6dd5f8e9e7c25ecd5ae4fcb4ad32e57786bb46e03`. You can check the full list of account addresses [here](https://github.com/xJonathanLEI/starkli/blob/e9a28f1b6e37bcc9fc53b7b7130e935894856739/src/account.rs#L76).
+
+To facilitate the seamless execution of Starkli commands, it is crucial to configure specific environment variables. Two primary variables are essential for this purpose:
+
+##### Account Configuration:
+
+Create an environment variable for your account, specifically for a pre-funded account on the local development network.
+
+```bash
+export STARKNET_ACCOUNT=katana-0
+```
+
+##### Network Designation:
+
+Establish another environment variable to specify the network, particularly targeting the local katana devnet.
+bash
+
+```bash
+export STARKNET_RPC=http://0.0.0.0:5050
+```
+
+### Contract Deployment and Interaction
+
+1. Create a Vote project
+
+```console
+scarb new vote
+```
+
+Add contract dependencies to scarb.toml
+
+```toml
+[dependencies]
+starknet = "2.5.0"
+[[target.starknet-contract]]
+```
+
+Copy the vote contract to lib.cairo
```

Comment:
This contract is interesting, but is huge IMHO.
What do you think about having a simple counter or something like this? Or perhaps you've some other cases you want to add?

Reply:
You are right. I suggest we can show a simple/basic example with a `counter contract`. Then we also can show an intermediate example with this `voting contract`

Reply:
Yep sounds good! The idea is as some reader may just dive into starknet, having something simpler first would not discourage them. :)